packages/orbit-components/src/primitives/IllustrationPrimitive/index.tsx

```diff
@@ -38,7 +38,8 @@ const baseURL = "//images.kiwi.com";
 
 const IllustrationPrimitive = ({
   name,
-  alt,
+  alt = "",
```

Comment:
We have the same problem here. We can't have `alt=""` and `role` undefined, because the default value for `role` will be `"img"`. So I think we should follow the same approach as for `Illustration`